reana.yaml

```diff
@@ -0,0 +1,18 @@
+version: 0.9.3
+inputs:
+  files:
+    - codes/analysis.py
+workflow:
+  type: serial
+  resources:
+    dask:
+      image: docker.io/coffeateam/coffea-dask-cc7:0.7.22-py3.10-g7f049
+  specification:
+    steps:
+      - name: mystep
+        environment: docker.io/coffeateam/coffea-dask-cc7:0.7.22-py3.10-g7f049
+        commands:
+        - python codes/analysis.py
+outputs:
```

Comment:
Please introduce also some behavioural tests for the histogram output file presence and for log messages.

```
all events 53446198
number of chunks 534
```